test/micro/org/openjdk/bench/javax/crypto/full/AESGCMBench.java

```diff
@@ -26,41 +26,38 @@
 import org.openjdk.jmh.annotations.Param;
 import org.openjdk.jmh.annotations.Setup;
 
-import javax.crypto.BadPaddingException;
 import javax.crypto.Cipher;
-import javax.crypto.IllegalBlockSizeException;
-import javax.crypto.NoSuchPaddingException;
 import javax.crypto.spec.GCMParameterSpec;
 import javax.crypto.spec.SecretKeySpec;
-import java.security.InvalidAlgorithmParameterException;
-import java.security.InvalidKeyException;
-import java.security.NoSuchAlgorithmException;
-import java.security.spec.InvalidParameterSpecException;
 
+/**
+ * This performance tests runs AES/GCM encryption and decryption using byte[]
+ * as input and output buffers for single and multi-part testing.
+ *
+ * This test rotates the IV and creates a new GCMParameterSpec for each encrypt
+ * benchmark operation
+ */
 
 public class AESGCMBench extends CryptoBase {
 
-    @Param({"AES/GCM/NoPadding"})
-    private String algorithm;
-
     @Param({"128"})
     private int keyLength;
 
-    @Param({""+16*1024})
+    @Param({"" + 1024, "" +  1500, "" + 4096, "" + 16384})
```

Comment:
Now that you don't calculate anymore, how about just string literals like `"1024"` etc?

Reply:
Funny.. I didn't even notice that